q2_types/genome_data/tests/test_format.py

```diff
@@ -59,44 +59,12 @@ def test_genes_dirfmt_fa_with_suffix(self):
 
         fmt.validate()
 
-    def test_genes_dirfmt_fa_with_prefix(self):
-        dirpath = self.get_data_path('genes-with-prefix')
-        fmt = GenesDirectoryFormat(dirpath, mode='r')
-
-        fmt.validate()
-
-    def test_genes_dirfmt_fa_with_wrong_prefix(self):
-        dirpath = self.get_data_path('genes-with-wrong-prefix')
-        fmt = GenesDirectoryFormat(dirpath, mode='r')
-
-        with self.assertRaisesRegex(
-                ValidationError,
-                'Missing one or more files for GenesDirectoryFormat'
-        ):
-            fmt.validate()
-
     def test_proteins_dirfmt_fa_with_suffix(self):
         dirpath = self.get_data_path('proteins-with-suffix')
```

Comment:
Can you also rename al those directories to not be anymore `*-with suffix` but rather just `*` (or something along those lines)? With the updated regex we're getting rid of the whole suffix/prefix concept.